deploy/examples/prod-caddy/.env.example

```
@@ -0,0 +1,25 @@
+# GlycemicGPT -- Production (Caddy) environment
+# Copy to .env and fill in ALL values before starting.
+
+# Image version (default: latest)
+# IMAGE_TAG=0.5.0
+
+# Database (REQUIRED)
+POSTGRES_USER=glycemicgpt
+POSTGRES_PASSWORD=  # Generate: openssl rand -hex 32
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)